web/src/views/DandisetLandingView/DandisetMain.vue

```diff
@@ -5,6 +5,14 @@
         {{ meta.name }}
       </h1>
     </v-row>
+    <v-row
+      class="mx-2"
+      align="center"
+    >
+      <v-col cols="auto">
+        DANDI: {{ meta.identifier }}
+      </v-col>
+    </v-row>
```

Comment:
@sanuann - move this section to before the previous row and replace `DANDI: {{ meta.identifier }}` with just ` {{ meta.id }}`